source/common/router/rds_impl.h

```diff
@@ -140,12 +140,11 @@ class RdsRouteConfigSubscription : Envoy::Config::SubscriptionCallbacks,
       const uint64_t manager_identifier,
       Server::Configuration::ServerFactoryContext& factory_context,
       ProtobufMessage::ValidationVisitor& validator, Init::Manager& init_manager,
-      const std::string& stat_prefix, RouteConfigProviderManagerImpl& route_config_provider_manager,
-      const envoy::api::v2::core::ConfigSource::XdsApiVersion xds_api_version =
-          envoy::api::v2::core::ConfigSource::AUTO);
+      const std::string& stat_prefix,
+      RouteConfigProviderManagerImpl& route_config_provider_manager);
 
   bool validateUpdateSize(int num_resources);
-  std::string loadTypeUrl();
+  static std::string loadTypeUrl(envoy::api::v2::core::ApiVersion resource_api_version);
```

Comment:
Is there definite reason for changing this method to static? This is from my bad implementation, I think that these `loadTypeUrl` which is scattered all of xDS implementation should be aggregated like `sotwGrpcMethod()`. If you agree with my opinion, I will fix them with other PR.

Reply:
I made them static as they don't really depend on object state for the most part, they are helper functions that we can use in the constructor based on the presented config source.

Reply:
ok, looks good for me